addons/auth_signup/data/mail_template_data.xml

```diff
@@ -341,5 +341,99 @@
         </record>
 
+        <!-- Email template for new device login alert -->
+        <record id="alert_new_device_email" model="mail.template">
+            <field name="name">Settings: Alert New Device</field>
+            <field name="model_id" ref="base.model_res_users"/>
+            <field name="subject">New Connection to your Account</field>
+            <field name="email_from">"{{ object.company_id.name }}" &lt;{{ (object.company_id.email or user.email) }}&gt;</field>
+            <field name="email_to">{{ object.email_formatted }}</field>
+            <field name="description">Sent to user after his account got logged into from an unknown device</field>
+            <field name="body_html" type="html">
+<table border="0" cellpadding="0" cellspacing="0" style="padding-top: 16px; background-color: #FFFFFF; font-family:Verdana, Arial,sans-serif; color: #454748; width: 100%; border-collapse:separate;"><tr><td align="center">
+<table border="0" cellpadding="0" cellspacing="0" width="590" style="padding: 16px; background-color: #FFFFFF; color: #454748; border-collapse:separate;">
+<tbody>
+    <!-- HEADER -->
+    <tr>
+        <td align="center" style="min-width: 590px;">
+            <table border="0" cellpadding="0" cellspacing="0" width="590" style="min-width: 590px; background-color: white; padding: 0px 8px 0px 8px; border-collapse:separate;">
+                <tr><td valign="middle">
+                    <span style="font-size: 10px;">New Sign-In With Your Account</span><br/>
+                    <span style="font-size: 20px; font-weight: bold;">
+                        <t t-out="object.name or ''">Marc Demo</t>
+                    </span>
+                </td><td valign="middle" align="right">
+                    <img t-attf-src="/logo.png?company={{ object.company_id.id }}" style="padding: 0px; margin: 0px; height: auto; width: 80px;" t-att-alt="object.company_id.name"/>
+                </td></tr>
+                <tr><td colspan="2" style="text-align:center;">
+                  <hr width="100%" style="background-color:rgb(204,204,204);border:medium none;clear:both;display:block;font-size:0px;min-height:1px;line-height:0; margin: 16px 0px 16px 0px;"/>
+                </td></tr>
+            </table>
+        </td>
+    </tr>
+    <!-- CONTENT -->
+    <tr>
+        <td align="center" style="min-width: 590px;">
+            <table border="0" cellpadding="0" cellspacing="0" width="590" style="min-width: 590px; background-color: white; padding: 0px 8px 0px 8px; border-collapse:separate;">
+                <tr><td valign="top" style="font-size: 13px;">
+                    <div>
+                        Dear <t t-out="object.name or ''">Marc Demo</t>,<br/><br/>
+                        A new device has logged into your account.<br/><br/>
+                        When: <t t-out="ctx['date']">day, month dd, yyyy - hh:mm:ss (GMT)</t><br/>
+                        Where: <t t-out="ctx['location']">City, Region, Country</t><br/>
```

Comment:
```suggestion
                        <t t-if="ctx['location']">
                        Location: <t t-out="ctx['location']">City, Region, Country</t><br/>
                        </t>
```
instead of diplaying weird "Where: unknow", you can only show info when you actually have it (and d the same for the other fields too)